src/Fixer/Basic/BracesFixer.php

```diff
@@ -297,12 +303,12 @@ function ($item) {
                     || !($nextToken->isWhitespace() && $nextToken->isWhitespace(" \t"))
                     && substr_count($nextToken->getContent(), "\n") === 1 // preserve blank lines
```

Comment:
lets keep one standard in the project and create a Fixer to let people run there own CS rules on own projects :)